jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -1629,9 +1629,23 @@
         frequency: 'H * * * *' # At least every hour
         json: 1
         trigger-job: 'ci-kubernetes-build'
-    
-    ### kubernetes-e2e-upgrades-gke #maisem
 
+    ### etcd upgrades #mml
+    - kubernetes-e2e-gce-gci-latest-upgrade-etcd: # upgrade etcd from 2.2 to 3.0, using latest
+        job-name: ci-kubernetes-e2e-gce-gci-latest-upgrade-etcd
+        timeout: 110
```

Comment:
https://github.com/kubernetes/test-infra/pull/2008/ added `jenkins-timeout` key, so add a `jenkins-timeout: 210` for both jobs :-) (Things change fast!)